### PR TITLE
add tests for notification reducer

### PR DESCRIPTION
**Pull request by felipebrunetti94**

### CODE REVIEW

File: spec/javascripts/reducers/notification_spec.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/notifications";
+
+describe("Notification Reducer",() => {
+  it("should return initialState", () => {
```
Comment (nolleto) [inline spec/javascripts/reducers/notification_spec.js]: I don't know what is the pattern for the test's messages but I would remove the word "should" and use `describe` for each scenario and change the `it` message for what is the returning.

####  The word "should"

> used to indicate what is probable.
As the test result is certain and not likely I would not use "should".

#### Description

I would probably do something like this:

```js
  describe('DEFAULT', () => {
    it('returns an empty array', () => {
      expect(reducer(undefined, [])).toEqual([]);
    });
  })
  

  describe('ADD_NOTIFICATION', () => {
    it('adds a new notification into state', () => {
      const state = [{ id: 1 }, { id: 2 }];

      const action = {
        type: actionsTypes.ADD_NOTIFICATION,
        notification: { id: 3 }
      };
      
      // Also, I would recommend to you to write the whole expected object to keep your test more explicit.
      expect(reducer(state, action)).toEqual([{ id: 1 }, { id: 2 }, { id: 3 }]);
    });
  });
```

Comment (felipebrunetti94) [reply spec/javascripts/reducers/notification_spec.js]: thanks @nolleto , changing right now. If you have any articles or tips for writing better tests, I would be glad to read :D

File: spec/javascripts/reducers/notification_spec.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+describe("Notification Reducer",() => {
+  describe("DEFAULT", () => {
+    it("return the initialState", () => {
+      const initialState = [];
+      expect(reducer(undefined, initialState)).toEqual([]);
+    });
+  })
+
+describe('ADD_NOTIFICATION', () => {
```
Comment (talyssonoc) [inline spec/javascripts/reducers/notification_spec.js]: The indentation here is broken.

File: spec/javascripts/reducers/notification_spec.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    it('adds a notification to the state',() => {
```
Comment (talyssonoc) [inline spec/javascripts/reducers/notification_spec.js]: We should have the two cases here, right? The one for a single notification and the other for multiple notifications.

Comment (felipebrunetti94) [reply spec/javascripts/reducers/notification_spec.js]: Done!

**Pull request merged**